packages/react-i18n/src/babel-plugin/test/babel-plugin.test.ts

```diff
@@ -380,6 +380,122 @@ describe('babel-plugin-react-i18n', () => {
       );
     });
   });
+
+  describe('explicit paths', () => {
+    it('injects proper translation function for only fallback case', async () => {
+      expect(
+        await transformUsingI18nBabelPlugin(
+          useI18nFixture,
+          optionsForExplicitImportFile('MyComponent.tsx', 'fallbackOnly'),
+          {mode: 'with-explicit-paths'},
+        ),
+      ).toBe(
+        await normalize(
+          `import _en from './${TRANSLATION_DIRECTORY_NAME}/en.json';
+          import React from 'react';
+          import {useI18n} from '@shopify/react-i18n';
+
+          export default function MyComponent() {
+            const [i18n] = useI18n({
+              id: 'MyComponent_${defaultHash}',
+              fallback: _en,
+              translations(locale) {
```

Comment:
In this case can we immediately return? There doesn't seem to be much value in having these empty checks.

Would `translations(local) {return} be enough?

Reply:
translations is actually optional, so we can 🔥 the entire translate method when there is no additional lang other than default

Reply:
Michelle mentioned this as an overall improvement we can make since the `translations` function is not required: https://github.com/Shopify/quilt/pull/1973#discussion_r691377071. I agree with Michelle that it should be a separate PR since it's an overall general improvement. I can add it to my workload as something to do right after this PR.

Reply:
Cool, happy to do that in a follow up :)